drake/systems/plants/xmlUtil.cpp

```diff
@@ -207,14 +241,14 @@ string resolveFilename(const string& filename,
   } else {
     std::string normalized_root_dir = spruce::path(root_dir).getStr();
 
-    // if root_dir is a relative path then convert it to absolute
+// if root_dir is a relative path then convert it to absolute
 #ifdef _WIN32
-    bool dirIsRelative = !(normalized_root_dir.size() >= 2
-                           && std::isalpha(normalized_root_dir[0])
-                           && normalized_root_dir[1] == ':');
+    bool dirIsRelative = !(normalized_root_dir.size() >= 2 &&
+                           std::isalpha(normalized_root_dir[0]) &&
+                           normalized_root_dir[1] == ':');
 #else
-    bool dirIsRelative = !(normalized_root_dir.size() >= 1
-                           && normalized_root_dir[0] == '/');
+    bool dirIsRelative =
+        !(normalized_root_dir.size() >= 1 && normalized_root_dir[0] == '/');
```

Comment:
Put another way: if your code is clang-format clean (and doesn't look like it fell off a corner case and into a volcano), it will also be style-guide compliant, saving us all some nitpicking.  

That said, clang-format will also change some style-guide compliant formatting into other style-guide compliant formatting, as seen here.  I think clang-format-using authors should feel free to make such changes.  It would be a big hassle to run clang-format and then back out its changes on lines you didn't otherwise affect.